assembl/templates/includes/javascript.jinja2

```diff
@@ -5,7 +5,7 @@
   var urlArgs = null;
   {% endif %}
   var assembl_locale = '{{locale}}';
-  var raven_url = '{{raven_url}}';
+  var sentry_dsn = '{{sentry_dsn}}';
```

Comment:
I think you can remove this now.

Reply:
I think we need it for v1 js reporting (raven)

Reply:
You're right 👍